src/shared/components/common/paginator.tsx

```diff
@@ -23,6 +24,7 @@ export class Paginator extends Component<PaginatorProps, any> {
         <button
           className="btn btn-secondary"
           onClick={linkEvent(this, this.handleNext)}
+          disabled={this.props.nextDisabled || false}
```

Comment:
Is the `|| false` necessary here?

Reply:
Good catch, it's a residue after nextDisabled was optional to not pass "undefined" to the property, I'll fix it